flexget/utils/tools.py

```diff
@@ -428,20 +428,16 @@ def get_current_flexget_version():
     return flexget.__version__
 
 
-def parse_filesize(text_size, base2=False):
+def parse_filesize(text_size, base2=True):
     """
-    Expands a data size into its bytes as an integer i.e. '1 MB' becomes 1000000.
-    If base2 is specified then assume bibytes so i.e. '1 MB' becomes 1048576
+    Expands a data size into its bibytes as an integer i.e. '1 MB' becomes 1048576.
```

Comment:
bibytes aren't a thing :wink: There are bytes (which is what the function currently returns) and then there are kibi-, mebi-, gibibytes (in the base2 system.) We are thinking the return should be changed to return mebibytes though, since that is what we use in the content_size field. To achieve that we need to divide the return by 1024*1024, and update the docstring.